File: src/03_to_xerox/to_xerox.cpp
#include <iostream>
#include <algorithm>

using namespace std;

int main(void) {
    int n, k;
    cin >> n >> k;

    int i = max(n / k, 1);
    int result_1 = abs(n - k * i);
    i++;
    int result_2 = abs(n - k * i);
    cout << min(result_1, result_2) << endl;
    return 0;
}

File: src/03_to_xerox/to_xerox.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var n, k, i, res_1, res_2 int
	fmt.Scanf("%d %d", &n, &k)
	i = max(n / k, 1)
	res_1 = int(math.Abs(float64(n - k * i)))
	i++
	res_2 = int(math.Abs(float64(n - k * i)))
	fmt.Println(min(res_1, res_2))
}
